common/src/main/java/gov/cms/ab2d/common/httpclient/APIClient.java

```diff
@@ -0,0 +1,173 @@
+package gov.cms.ab2d.common.httpclient;
```

Comment:
Reusable `APIClient` makes sense, but can we keep it under `e2e` and make the load test import it?

Reply:
You could move it under 'src/test` in e2e, and then use https://maven.apache.org/plugins/maven-jar-plugin/examples/create-test-jar.html to produce a "test jar" so that the load test could import it as a dependency.

Reply:
Yeah I think that makes sense.